lib/groupify/adapter/active_record/group.rb

```diff
@@ -50,7 +44,10 @@ def merge!(source)
 
       module ClassMethods
         def with_member(member)
-          member.groups
+          #member.groups
+          joins(:group_memberships_as_group).
+          merge(Groupify.group_membership_klass.where(member_id: member.id, member_type: member.class.base_class.to_s)).
```

Comment:
@joelvh I couldn't quite understand what this change was for, because when I revert it back the tests still pass. From the commits, it looks like this adds support for querying on subclasses so that STI can be supported. This is a good example of where a specific test for this would be helpful.

I think this line can be simplified by changing to `merge(member.group_memberships_as_member)`